common/Reducers.jsm

```diff
@@ -39,7 +39,6 @@ const INITIAL_STATE = {
   },
   Sections: [],
   Pocket: {
-    isUserLoggedIn: false,
```

Comment:
We initially start out without a true or false state, we just don't know yet.

Reply:
actually could you make this `null` instead of false, just so we can see the shape of the state?